pelicanconf.py

```diff
@@ -1,3 +1,27 @@
 # -*- coding: utf-8 -*-
 
+AUTHOR = 'Komitywa'
+SITENAME = 'Komitywa.org'
+SITEURL = ''
```

Comment:
There is no need to configure it here, because `SITEURL` is not defined by default.